micromamba/tests/test_shell.py

```diff
@@ -62,8 +62,131 @@ def test_hook(tmp_home, tmp_root_prefix, shell_type):
     assert res["context"]["shell_type"] == shell_type
     assert set(res["actions"].keys()) == {"print"}
 
-    if shell_type != "cmd.exe":
-        assert res["actions"]["print"]
+    @classmethod
+    def teardown_method(cls):
+        os.environ["MAMBA_ROOT_PREFIX"] = TestShell.root_prefix
+        os.environ["CONDA_PREFIX"] = TestShell.current_prefix
+
+        if Path(TestShell.root_prefix).exists():
+            shutil.rmtree(TestShell.root_prefix)
+
+    @pytest.mark.parametrize(
+        "shell_type",
+        [
+            "bash",
+            "posix",
+            "powershell",
+            "cmd.exe",
+            "xonsh",
+            "zsh",
+            "fish",
+            "tcsh",
+            "nu",
+        ],
+    )
+    def test_hook(self, shell_type):
```

Comment:
Thanks a lot for adding these tests! How much work is it to also have them run with Mamba?